CRM/Utils/String.php

```diff
@@ -1034,6 +1034,11 @@ public static function parseOneOffStringThroughSmarty($templateString) {
     $cachingValue = $smarty->caching;
     set_error_handler([$smarty, 'handleSmartyError'], E_USER_ERROR);
     $smarty->caching = 0;
+    if (CRM_Utils_Constant::value('SMARTY_DEBUG_STRINGS')) {
```

Comment:
@eileenmcnaughton do we need a specific one for smarty here why not just like the general debugging flag?

Reply:
@seamuslee001 yeah could do - it could get pretty noisy?